src/utils/dateUtils.ts

```diff
@@ -0,0 +1,12 @@
+const ONE_WEEK_MILLIS = 7 * 24 * 60 * 60 * 1000;
+
+export const calculateWeeksSinceDate = (date: Date): number => {
+  const now = new Date();
+  return getWeeksBetween(new Date(date), now);
```

Comment:
Fordi vår mock-data hentes fra json-filer og ikke typede ts-filer, får vi her inn en string-dato lokalt.
Jeg er litt imot mockdataoppsettet vårt når det fører til at vi må "hacke" til funksjoner for å funke lokalt (i tillegg til at man må kjøre `npm run start` på nytt når man endrer mock-data da).

Reply:
Kanskje vi kan begynne å flytte noe til ts-filer? Eller hvertfall gjøre det sånn i neste app?

Reply:
Vi gjorde vel en sjau på å flytte fra json til .ts-filer i [syfomodiaperson ](https://github.com/navikt/syfomodiaperson) for en stund tilbake, så hadde vært fint å få gjort tilsvarende her. Evt flytte gradvis når man er inne og endrer på ting hvis det er mulig.

Reply:
Jeg kan evt sette opp et kort for det i Trello 👍🏼